targets command does not work

On current "dev" branch head.

This also affects VSCode plugin, can not list and select target.

```
$ tinygo targets
...
esp32-mini32
esp32
panic: duplicate value 'esp32c3' in field : BuildTags

goroutine 1 [running]:
github.com/tinygo-org/tinygo/compileopts.(*TargetSpec).overrideProperties(0xc000374000, 0xc00036db80)
...
```

--- a/Makefile
+++ b/Makefile
@@ -528,6 +528,8 @@ endif
 	@$(MD5SUM) test.hex
 	$(TINYGO) build -size short -o test.hex -target=feather-m4          examples/blinky1
 	@$(MD5SUM) test.hex
+	$(TINYGO) build -size short -o test.hex -target=matrixportal-m4     examples/blinky1
+	@$(MD5SUM) test.hex
 	$(TINYGO) build -size short -o test.hex -target=pybadge             examples/blinky1
 	@$(MD5SUM) test.hex
 	$(TINYGO) build -size short -o test.hex -target=metro-m4-airlift    examples/blinky1
@@ -690,7 +692,9 @@ ifneq ($(XTENSA), 0)
 	$(TINYGO) build -size short -o test.bin -target mch2022             examples/serial
 	@$(MD5SUM) test.bin
 endif
-	$(TINYGO) build -size short -o test.bin -target=esp32c3           	examples/serial
+	$(TINYGO) build -size short -o test.bin -target=esp32c3             examples/serial
+	@$(MD5SUM) test.bin
+	$(TINYGO) build -size short -o test.bin -target=esp32c3-12f         examples/serial
 	@$(MD5SUM) test.bin
 	$(TINYGO) build -size short -o test.bin -target=m5stamp-c3          examples/serial
 	@$(MD5SUM) test.bin

--- a/targets/esp32c3-12f.json
+++ b/targets/esp32c3-12f.json
@@ -1,5 +1,5 @@
 {
 	"inherits": ["esp32c3"],
-	"build-tags": ["esp32c312f", "esp32c3", "esp"]
+	"build-tags": ["esp32c312f"]
 }
 

--- a/targets/matrixportal-m4.json
+++ b/targets/matrixportal-m4.json
@@ -1,6 +1,6 @@
 {
     "inherits": ["atsamd51j19a"],
-    "build-tags": ["atsamd51j19a", "matrixportal_m4"],
+    "build-tags": ["matrixportal_m4"],
     "serial": "usb",
     "serial-port": ["acm:239a:80c9", "acm:239a:00c9", "acm:239a:80ca"],
     "flash-1200-bps-reset": "true",

